client/src/renderer/components/Editor/Span.jsx

```diff
@@ -0,0 +1,15 @@
+/** @format */
+
+import { h } from "preact";
+import { renderDocumentNodes } from "dom/render";
+
+/**
+ * Constructs a span within the editor from a DOM node.
+ * @param {Object} props Component properties.
+ * @param {Object} props.node DOM Node used to create a paragraph.
+ * @return {PreactElement} A rendered preact element.
+ */
+export default function Paragraph(props) {
```

Comment:
Please make sure the class name is the same as the file. I think this is supposed to be like this:

```suggestion
export default function Span(props) {
```

Reply:
Nice catch, I forgot to change that out of excitement that it worked it seems. Resolved by both 9f5bfd8 and fd4820a.